doc/users/next_whats_new/aria.rst

```diff
@@ -0,0 +1,23 @@
+All ``Arist`` now carry wai-aria data
```

Comment:
```suggestion
All ``Artist`` now carry wai-aria data
```

spelling

Reply:
resolving in favor of @QuLogic's version which also adds the character to the underline (which is also needed for rst)